TLM/TLM/Manager/Impl/TrafficPriorityManager.cs

```diff
@@ -425,9 +425,10 @@ public class TrafficPriorityManager
             }
 
             if ((vehicle.m_flags & Vehicle.Flags.Spawned) == 0) {
-                Log.WarningIf(
-                    logPriority,
-                    () => $"TrafficPriorityManager.HasPriority({vehicleId}): Vehicle is not spawned.");
+                if (logPriority) {
```

Comment:
Can't we do a [Conditional("DEBUG")] for Log.Warning instead?

Reply:
nope, because in this case it's not `DebugWarning` and `if` is to defer string format boxing when it's actually used :)